README.md

```diff
@@ -246,55 +282,114 @@ plugin, please consider one of the following options:
 - To use a config file located outside the package/project directory, a config 
   file may be added to that directory specifying a parent config path to the 
   other config file, e.g., `parent_config: path/to/.swiftlint.yml`.
-- You can also consider the use of a Run Script Build Phase in place of the
-  build tool plugin.
+- You can also consider the use of a
+  [Run Script Build Phase](#xcode-run-script-build-phase) in place of the build
+  tool plugin.
 
-#### Xcode Projects
+### Xcode Run Script Build Phase
 
-You can integrate SwiftLint as an Xcode Build Tool Plugin if you're working
-with a project in Xcode.
+If the build tool plugin does not work for your project setup or when
+additional custom setup is required, SwiftLint can be added as a Run Script
+Build Phase. This is useful when a project setup relies on the `--config`
+SwiftLint option or to lint all targets together in a single `swiftlint`
+invocation. File inclusions and exclusions can be configured in the
+[`.swiftlint.yml` configuration](#configuration).
 
-Add SwiftLint as a package dependency to your project without linking any of the
-products.
+> [!IMPORTANT]
```

Comment:
Please place this important callout immediately following the `Uncheck Based on dependency analysis` tip.